lib/index.js

```diff
@@ -178,6 +136,15 @@ Namespace.prototype.getNormalizedText = function() {
     return this._siteInfo.namespaces[this._id + '']['*'].replace(/ /g, '_');
 };
 
+/**
+ * Are subpages allowed for this namespace?
+ *
+ * @returns {boolean}
+ */
+Namespace.prototype.subpagesAllowed = function() {
+    return this._siteInfo.namespaces[this._id + ''].subpages !== undefined;
+};
+
```

Comment:
In `RESTBase` to decrease the memory footprint of the `siteInfo` cache we delete all the properties that are not required for the module to work. A list of required properties is documented in the [SiteInfo](https://github.com/wikimedia/mediawiki-title/blob/master/lib/index.js#L16-L36) object documentation. Could you please add the `subpages` field there?